tests/integration/test_gitlab.py

```diff
@@ -97,6 +97,25 @@ def test_parent_project(self):
         assert self.project.get_fork().parent.namespace == "packit-service"
         assert self.project.get_fork().parent.repo == "ogr-tests"
 
+    def test_get_owners(self):
+        owners = self.project.get_owners()
+        assert ["lachmanfrantisek", "lbarcziova"] == owners
```

Comment:
What if `owners` is not `["lachmanfrantisek", "lbarcziova"]`, but `["lbarcziova", "lachmanfrantisek"]`?
```python
assert {"lachmanfrantisek", "lbarcziova"} == set(owners)
```